README.md

```diff
@@ -1 +1,523 @@
+
+## 1.2 Foundational Authority
+
+IPVM workflows are built on top of cryptographic capabilities, providing a strong basis for distributed computation in trustless networks. This even provides a clear basis for crossing Web 2.0 and Web3 systems, other computation networks, local operation (in full or part).
+
+The IPVM Workflow spec extends on several other specs that have been developed to provide this basis:
+
+```
+┌───────────────────────────────────────────────┬───────────────────────────┐
+│                                               │                           │
+│             Human Configuration:              │                           │
+│  Defaults, Exception Handling, Comments, Tags │                           │
+│               (IPVM Workflow)                 │                           │
+│                                               │  Multi-Request Pipelining │
+├───────────────────────────────────────────────┤     (UCAN Invocation)     │
+│                                               │                           │
+│      IPVM Config, Verification Level, etc     │                           │
+│                 (IPVM Task)                   │                           │
+│                                               │                           │
+├───────────────────────────────────────────────┴───────────────────────────┤
+│                                                                           │
+│                                 Call Graph                                │
+│                             (UCAN Invocation)                             │
+│                                                                           │
+├───────────────────────────────────────────────────────────────────────────┤
+│                                                                           │
+│                                 Authority                                 │
+│                                (UCAN Core)                                │
+│                                                                           │
+└───────────────────────────────────────────────────────────────────────────┘
+```
+
+# 2 Envelope
+
+The outer wrapper of a workflow MUST contain the following fields:
+
+| Field           | Type       | Description                                                             | Required |
+|-----------------|------------|-------------------------------------------------------------------------|----------|
+| `ipvm/workflow` | `Workflow` | IPVM Workflow                                                           | Yes      |
+| `signature`     | `VarSig`   | [VarSig](https://github.com/ChainAgnostic/varsig/) of serialized fields | Yes      |
```

Comment:
Seems like the repo is still WIP for VarSig? Can we get a footnote about it?

Reply:
We're trying to retcon [varsigs](https://github.com/multiformats/multicodec/blob/master/table.csv#L493) a bit by adding more data to them. I hope to get a spec out in the next day or two.